tests/Doctrine/Tests/Persistence/Mapping/DriverChainTest.php

```diff
@@ -115,7 +118,7 @@ public function testDefaultDriver(): void
         self::assertNull($chain->getDefaultDriver());
 
         $chain->setDefaultDriver($defaultDriver);
-        $chain->addDriver($companyDriver, 'Doctrine\Tests\Models\Company');
+        $chain->addDriver($companyDriver, 'Doctrine\Tests\Persistence\Mapping\Fixtures\Manager');
```

Comment:
Why not use `::class` here?

Reply:
Because it's a namespace:

https://github.com/doctrine/persistence/blob/fdef2dd7da053bc97841fb45b866a7756bfe0f2f/lib/Doctrine/Persistence/Mapping/Driver/MappingDriverChain.php#L49-L59